templates/named.conf.local.forward.j2

```diff
@@ -0,0 +1,17 @@
+## {{ ansible_managed }}
+{% for fwd_zone in bind_config_forward_zones %}
+######## {{ fwd_zone.name }} ({{ fwd_zone.zones|count }} zones)
+{% for zone in fwd_zone.zones %}
+zone "{{ zone }}" {
+    type forward;
+{% if fwd_zone.forward is defined %}
+    forward {{ fwd_zone.forward }}; 
```

Comment:
Okay, commit 718b50ef0ac6b247f197cb2e3c478e3111fa66b7 pushed to my branch which defaults to `forward only` if it's not specified for the zones